reporter/jsonConsole.go

```diff
@@ -200,6 +207,10 @@ func (c *jsonConsole) StepStart(stepText string) {
 }
 
 func (c *jsonConsole) StepEnd(step gauge.Step, res result.Result, execInfo gm.ExecutionInfo) {
+	si := new(stepInfo)
```

Comment:
Simplify this to `&stepInfo{step: &step, protoStep: res.(*result.StepResult).Item().(*gm.ProtoStep)}`